dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -0,0 +1,70 @@
+class LtiV1Controller < ApplicationController
+  # Don't require an authenticity token because LTI Platforms POST to this controller.
+  skip_before_action :verify_authenticity_token
+
+  # [GET/POST] /lti/v1/login(/:platform_id)
+  def login
+    if params[:client_id]
+      query_params = {client_id: params[:client_id], issuer: params[:iss]}
```

Comment:
Is it acceptable if only `client_id` parameter is provided without `iss` parameter?

Reply:
I'll add some light docs as comments in this controller, and a link to the LTI 1.3 spec. The `iss` parameter will always be present in the request, as it's required by LTI 1.3.

Reply:
I think we should add a check for this:
```suggestion
    if params[:client_id] && params[:iss]
      query_params = {client_id: params[:client_id], issuer: params[:iss]}
```
OR
```suggestion
    if params[:client_id]
      query_params = {client_id: params[:client_id], issuer: params.require(:iss)}
```

`params.require(:iss)` will raise `ActionController::ParameterMissing` if `iss` parameter isn't provided with `client_id` parameter

Reply:
We don't need to check for `iss`, it is required that an LTI 1.3 Platform include it in the Initiate Login request https://www.imsglobal.org/spec/security/v1p0/#step-1-third-party-initiated-login. If it's not present, that Platform isn't LTI 1.3 compliant, and we won't integrate with it.